ladder/teams/operator.yaml

```diff
@@ -1,10 +1,5 @@
 members:
 - aanm
 - christarazi
-- dlapcevic
 - joamaki
-- nebril
```

Comment:
I know that Maciej has just gone on leave for the next couple of months but he plans to return. It's a bit of an inopportune time to bring this up given his focus is elsewhere currently.

Reply:
I'd say that for now with respect to Maciej's team membership, the best option would be to keep him in the teams and then we can discuss it with him when he is back from leave. I don't want to create the expectation that he is paying close attention to his GitHub notifications at this time and then be surprised about this when he gets back. For now, he is marked as "busy" on GitHub so reviews in these teams will not be assigned to him. This means he won't be inhibiting the review process during his time away.